b9lab-content/3-main-concepts/10-multistore-keepers.md

```diff
@@ -108,27 +108,357 @@ The AnteHandler is theoretically optional, but still a very important component
 
 BaseApp holds an anteHandler as a parameter that is initialized in the application's constructor. The most widely used anteHandler is the auth module.
 
-More info: https://github.com/cosmos/cosmos-sdk/blob/master/docs/basics/gas-fees.md#antehandler + https://github.com/cosmos/cosmos-sdk/blob/master/docs/basics/gas-fees.md 
+More info: https://github.com/cosmos/cosmos-sdk/blob/master/docs/basics/gas-fees.md#antehandler + https://github.com/cosmos/cosmos-sdk/blob/master/docs/basics/gas-fees.md
```

Comment:
Don't think we need to have two links to the same article. Would also suggesting adding the antihandler doc on docs.cosmos.network

```suggestion
More info: https://docs.cosmos.network/master/modules/auth/03_antehandlers.html + https://github.com/cosmos/cosmos-sdk/blob/master/docs/basics/gas-fees.md#antehandler
```

Reply:
Updated.